rapidpro.js

```diff
@@ -53,10 +53,9 @@ module.exports = function (config) {
     })
   }
 
-  const getContacts = function (groupUUID, callback) {
-    let url = contactsURL(groupUUID)
+  const getContacts = function (url, callback) {
```

Comment:
I think rather than passing in a URL here we should rather keep it how it was before and rather add the paging parameters as a parameter to this call. We could make the parameter optional by doing something like this:

```
const getContacts = function (url, page, callback) {
  if (typeof page === 'function') {
    callback = page
    page = null
  }
```